thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -77,7 +77,7 @@ export default Ember.Controller.extend({
       invisibleUrns: new Set(),
       hoverUrns: new Set(),
       filteredUrns: new Set(),
-      activeTab: ROOTCAUSE_TAB_DIMENSIONS,
+      activeTab: ROOTCAUSE_TAB_METRICS,
```

Comment:
Is this intentional - metrics as the default view? afaik the heatmap should be the first thing

Reply:
Selene suggested to move the metrics tab to the first tab (see task spreadsheet). If it's the first tab, then it should be the default one that's shown.